ref/Castle.Core-net45.cs

```diff
@@ -1582,7 +1582,7 @@ public abstract class XmlNodeBase : Castle.Components.DictionaryAdapter.IVirtual
         public Castle.Components.DictionaryAdapter.Xml.IXmlNamespaceSource Namespaces { get; }
         public Castle.Components.DictionaryAdapter.Xml.IXmlNode Parent { get; }
         public virtual Castle.Components.DictionaryAdapter.Xml.CompiledXPath Path { get; }
-        public event System.EventHandler Realized;
+        public  virtual  event System.EventHandler Realized;
```

Comment:
This looks like an unexpected change.

Reply:
... but correct. `Realized` is in fact declared `virtual`.

(Sorry, my summary of the changes to our `ref/` files was indeed incomplete, I should've made that list more complete.)

Reply:
Haha, yer it is. I've never written code like that before.